pkg/cmd/config/config_keys.go

```diff
@@ -0,0 +1,65 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/dcos/dcos-cli/api"
+	"github.com/spf13/cobra"
+)
+
+// newCmdConfigKeys returns the keys that a user can set in a configuration file.
+func newCmdConfigKeys(ctx api.Context) *cobra.Command {
+	var quietOutput bool
+	cmd := &cobra.Command{
+		Use:   "keys",
+		Short: "Print all the keys that can be set in a configuration file",
+		Args:  cobra.NoArgs,
+		Run: func(cmd *cobra.Command, args []string) {
+			keys := []string{
+				"core.dcos_acs_token",
+				"core.dcos_url",
+				"core.mesos_master_url",
+				"core.pagination",
+				"core.ssl_verify",
+				"core.timeout",
+				"core.ssh_user",
+				"core.ssh_proxy_ip",
+				"core.reporting",
+				"core.prompt_login",
+				"cluster.name",
+				"job.url",
+				"job.service_name",
+				"marathon.url",
+				"package.cosmos_url",
+			}
+
+			configMap := map[string]string{
```

Comment:
Could this map be returned by the config package (`config.Keys()`)? As we already have constants there.